src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const booksSlice = createSlice({
+  name: 'addBooks',
+  initialState: [],
```

Comment:
You did well with this milestone but just a little change to be implemented for you to get to the next milestone 😄 . Kindly Creates an initialState with a state value books that contains an empty array, just like you did for the categories.

Something like this  ` 
initialState: {
    books: [],
  },
`